kubectl-plugin/uninstall/root.go

```diff
@@ -0,0 +1,101 @@
+package uninstall
+
+import (
+	"context"
+	"fmt"
+	"github.com/kubesphere-sigs/ks/kubectl-plugin/common"
+	kstypes "github.com/kubesphere-sigs/ks/kubectl-plugin/types"
+	component2 "github.com/kubesphere-sigs/ks/kubectl-plugin/uninstall/component"
+	"github.com/spf13/cobra"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/kubernetes"
+	"strconv"
+)
+
+// NewUninstallCmd returns the command of uninstall kubesphere
+func NewUninstallCmd() (cmd *cobra.Command) {
+	opt := &uninstallOption{}
+	cmd = &cobra.Command{
+		Use:     "uninstall",
+		Short:   "Uninstall KubeSphere",
```

Comment:
I guess this command only supports uninstalling the components of KubeSphere. The description might be incorrect.

Reply:
All right, already modified.